x-pack/platform/plugins/shared/fleet/public/applications/fleet/sections/settings/components/settings_page/advanced_section.tsx

```diff
@@ -134,95 +137,132 @@ export const AdvancedSection: React.FunctionComponent<{}> = ({}) => {
+                  defaultMessage="Migrate to space-aware agent policies"
+                />
+              </h3>
+            }
+            description={
+              <p>
+                <FormattedMessage
+                  id="xpack.fleet.settings.advancedSection.switchLabel"
+                  defaultMessage="Take advantage of improved isolation and management by enabling space-specific agent policies."
+                />
+              </p>
+            }
+          >
+            <EuiFormRow>
+              <EuiButton
+                onClick={onClickEnableSpaceAwareness}
+                isLoading={mutateSpaceAwarenessIsLoading}
+              >
+                <FormattedMessage
+                  id="xpack.fleet.settings.deleteUnenrolledAgentsLabel"
+                  defaultMessage="Start migration"
+                />
+              </EuiButton>
+            </EuiFormRow>
+          </EuiDescribedFormGroup>
         </>
       )}
+
       <EuiSpacer size="m" />
-      <EuiForm component="form">
-        <EuiDescribedFormGroup
-          title={
-            <h3>
-              <FormattedMessage
-                id="xpack.fleet.settings.deleteUnenrolledAgentsLabel"
-                defaultMessage="Delete unenrolled agents"
-              />
-            </h3>
-          }
-          description={
-            <p>
-              <FormattedMessage
-                id="xpack.fleet.settings.advancedSection.switchLabel"
-                defaultMessage="Switching on this setting will enable auto deletion of unenrolled agents. For more information see our {docLink}."
-                values={{
-                  docLink: (
-                    <EuiLink target="_blank" external href={docLinks.links.fleet.settings}>
-                      <FormattedMessage
-                        id="xpack.fleet.settings.advancedSection.link"
-                        defaultMessage="docs"
-                      />
-                    </EuiLink>
-                  ),
-                }}
-              />
-            </p>
-          }
-        >
-          <EuiFormRow label="">
-            <EuiToolTip
-              content={
-                isPreconfigured
-                  ? i18n.translate('xpack.fleet.settings.advancedSection.preconfiguredTitle', {
-                      defaultMessage: 'This setting is preconfigured and cannot be updated.',
-                    })
-                  : undefined
+
+      <EuiDescribedFormGroup
+        title={
+          <h3>
+            <FormattedMessage
+              id="xpack.fleet.settings.deleteUnenrolledAgentsLabel"
+              defaultMessage="Delete unenrolled agents"
+            />
+          </h3>
+        }
+        description={
+          <p>
+            <FormattedMessage
+              id="xpack.fleet.settings.advancedSection.switchLabel"
+              defaultMessage="Switching on this setting will enable auto deletion of unenrolled agents. For more information see our {docLink}."
```

Comment:
```suggestion
              defaultMessage="Switching on this setting will enable auto-deletion of unenrolled agents. For more information, refer to the {docLink}."
```
(we avoid using “see” to improve accessibility)